src/gui/SearchHelpWidget.ui

```diff
@@ -235,6 +235,13 @@
           </property>
          </widget>
         </item>
+        <item row="3" column="1">
+         <widget class="QLabel" name="label_16">
+          <property name="text">
+           <string notr="true">group (g)</string>
```

Comment:
If you changed https://github.com/keepassxreboot/keepassxc/pull/4705/files#diff-957275f57754b628b5d0839a944bb19eR222 to only "group" it shouldn't say "(g)", right?

```suggestion
           <string notr="true">group</string>
```

Reply:
Look at all the other terms... the preferred shortened term is displayed in parenthesis.

Reply:
![image](https://user-images.githubusercontent.com/2809491/81420738-65221400-911e-11ea-9205-b04761cb8381.png)

Reply:
yeah. But wouldn't it need a `{QStringLiteral("g"), Field::Group}` line?

Reply:
I removed that already